pack/nodejs/helm.ts

```diff
@@ -78,6 +82,81 @@ export namespace v2 {
                 overrides.removeCallback()
             }
         }
+
+        parseTemplate(
+            yamlStream: string,
+            config: ChartOpts
+        ): { [key: string]: pulumi.CustomResource } {
+            const objs = jsyaml
+                .safeLoadAll(yamlStream)
+                .filter(a => a != null && "kind" in a)
```

Comment:
What does Helm do for resources that are missing `kind`?

Reply:
There is no such thing as a resource that does not have a `kind`.

Reply:
This is basically to filter out empty YAML objects, which Helm can produce lots of.